server/routes/avatarproxy.ts

```diff
@@ -1,15 +1,25 @@
 import ImageProxy from '@server/lib/imageproxy';
 import logger from '@server/logger';
+import { getHostname } from '@server/utils/getHostname';
 import { Router } from 'express';
 
 const router = Router();
 
 const avatarImageProxy = new ImageProxy('avatar', '');
 // Proxy avatar images
 router.get('/*', async (req, res) => {
-  const imagePath = req.url.startsWith('/') ? req.url.slice(1) : req.url;
-
+  let imagePath = '';
   try {
+    const jellyfinAvatar = req.url.match(
+      /(\/Users\/\w+\/Images\/Primary\/?\?tag=\w+&quality=90)$/
+    )?.[1];
+    if (!jellyfinAvatar) {
+      throw new Error('Provided URL is not a Jellyfin avatar.');
```

Comment:
what would you say towards getting the `mediaServerType` here when the error is being thrown? It would be less confusing to the users?

Reply:
Yep